app/templates/src/index.adoc

```diff
@@ -0,0 +1,54 @@
+= <%= title %>
+:!sectids:
+:imagesdir: images
+:source-highlighter: coderay
+:coderay-css: style
+
+== Images
+image::bespoke-logo.jpg[]
+
+== Keyboard Interaction
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-keys[bespoke-keys]
+
+== Touch Interaction
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-scale[bespoke-scale]
+
+== Bullet List
+[discrete]
+=== Powered by https://github.com/markdalgleish/bespoke-bullets[bespoke-bullets]
+[%build]
+* Bullet 1
+* Bullet 2
+* Bullet 3
+
+// No multimedia plugin usage?
+
+== Syntax Highlighting
+[source,js]
+----
+(function() {
+  function hanoi(n, a, b, c) {
+    return n ? hanoi(n - 1, a, c, b)
+        .concat([[a, b]])
+        .concat(hanoi(n - 1, c, b, a)) : [];
+  }
+  return hanoi(3, 'left', 'right', 'mid')
+    .map(function(d) {
+      return d[0] + ' -> ' + d[1];
+    });
+})();
+----
+
+// No named route plugin usage?
+
+== Just the beginning...
+[%build]
+* Edit http://asciidoctor.org/[AsciiDoc] markup in _'src/index.adoc'_
+* Edit http://stylus-lang.com/[Stylus] styles in _'src/styles/main.styl'_
+* Edit JavaScript in _'src/scripts/main.js'_
+
+[%build]
```

Comment:
One minor change. This option needs to be changed to a role (as shown below) since paragraphs don't currently support the build option.

```
[.build]
```